release-workflow.md

```diff
@@ -0,0 +1,58 @@
+# Workflow for Releasing Different Versions
+
+We release the latest versions of the main repository including the content from _The Turing Way_ book on Zenodo (DOI for all versions: [10.5281/zenodo.3233853](https://doi.org/10.5281/zenodo.3233853)).
+In this document, we describe the process for making a release on Zenodo through GitHub.
+
+We don't describe how to connect a GitHub account/repository to a Zenodo account.
+To learn more about that, please read the details in [the GitHub documentation](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content).
+
+## Updating release information
+
+- Update CITATION.cff file:
+  - Update `version: 1.0.1` (currently in line 7), `date-released: "2021-11-10"` (currently in line 13)
+- Validate the file locally on your computer using `cffconvert` (here we assume that the user knows how to use their terminal)
```

Comment:
Maybe we can refer here to the section on .cff that Faruk/Abel are trying to add right now?

Reply:
Good point, We can make the link from their Pull Request. 🙌